### PR TITLE
Deprecated OpenStreetMapImageryProvider

### PR DESCRIPTION
**Pull request by TomPed**
Removed `OpenStreetMapImageryProvider` and `OpenStreetMapImageryProviderSpec`.

For #3222

### CODE REVIEW

**Comment by lilleyse** [comment]
Update `CHANGES.md` and add a breaking change for `OpenStreetMapImageryProvider`. Look at the listing under `1.17` as an example.

**Comment by TomPed** [comment]
Updated.

File: CHANGES.md [hunk 1]
```diff
@@ -8,6 +8,9 @@ Change Log
 
 ### 1.18 - 2016-02-01
 
+* Breaking changes
+    * Removed `OpenStreetMapImageryProvider`
```
Comment (lilleyse) [inline CHANGES.md]: In following the others, just add another sentence: "Use `createOpenStreetMapImageryProvider` instead."
Also remove the white space.

**Comment by TomPed** [comment]
Updated.

**Comment by pjcozzi** [comment]
@TomPed when you open a pull request, please link to the issue (#3222 in this case) in the description.  For more details on opening a PR, see https://github.com/AnalyticalGraphicsInc/cesium/blob/master/CONTRIBUTING.md#opening-a-pull-request

**Comment by lilleyse** [comment]
Merged.

**Pull request merged**